src/encode/pattern/mod.rs

```diff
@@ -443,6 +449,7 @@ impl<'a> From<Piece<'a>> for Chunk {
                     "f" | "file" => no_args(&formatter.args, parameters, FormattedChunk::File),
                     "L" | "line" => no_args(&formatter.args, parameters, FormattedChunk::Line),
                     "T" | "thread" => no_args(&formatter.args, parameters, FormattedChunk::Thread),
+                    "P" | "PID" => no_args(&formatter.args, parameters, FormattedChunk::PID),
```

Comment:
I think the long name would be just lower case `pid` here to match up with `mdc` below.